packages/pub/sentry_dart_plugin/2.2.1.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "Sentry Dart Plugin",
+    "canonical": "pub:sentry_dart_plugin",
+    "version": "2.2.1",
+    "package_url": "https://pub.dev/packages/sentry_dart_plugin",
+    "repo_url": "https://github.com/getsentry/sentry-dart-plugin",
+    "main_docs_url": "https://docs.sentry.io/platforms/flutter/upload-debug/"
```

Comment:
do we need a created_at?

Reply:
wasn't sure if they initial registry needs this / if it's okay to just pick the today's date

Reply:
not sure about requirements, but we can just use current date yep